studio/src/constants.js

```diff
@@ -0,0 +1,22 @@
+export const CHECKOUT_CTA_TEXTS = {
```

Comment:
how about getting these values from tagging, and using it in both OST and rte field?

Reply:
may be it can still wait a bit here, no ?

Reply:
yeah, i'd like to discuss that separately before we do. having a tag for each possible buy now link text might be overwhelming, we need to think it through in peace (but CCD needs some labels now)

Reply:
👍